images/metering-ansible-operator/roles/meteringconfig/tasks/configure_storage.yml

```diff
@@ -195,3 +195,20 @@
       status: "True"
       message: "Finished configuring Hive storage"
       lastTransitionTime: "{{ now(utc=False).isoformat() + 'Z' }}"
+
+- name: Log Events for Configuring Storage
+  k8s_event:
+    state: present
+    name: Configuring Storage Event
+    namespace: "{{ meta.namespace }}"
+    message: Configure Storage
+    reason: Created
+    reportingComponent: Reporting components
+    type: Normal
+    source:
+      component: Metering components
+    involvedObject:
+      apiVersion: metering.openshift.io
+      kind: MeteringConfig
+      name: storage
+      namespace: "{{ meta.namespace }}"
```

Comment:
Same thing here with updating the `involvedObject.Name`, but let's also add a trailing newline to line 214.